test/blockdiagonal.jl

```diff
@@ -90,4 +90,13 @@ using Test
         @test Matrix(b1) ≈ b1
         @test b1 ≈ Matrix(b1)
     end  # Equality
+
+    @testset "Non-Square Matrix" begin
+		A1 = ones(2,4);
```

Comment:
Please can you change this code to match the style in the rest of the file?
- 4 spaces for indents
- spaces around operators and after commas
- no unnecessary `;`